#!/usr/bin/env python

PACKAGE = "wheel_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("front_left", double_t, 0, "front left wheel speed multiplier", 1 , -1, 1)
gen.add("front_right", double_t, 0, "front right wheel speed multiplier", 1 , -1, 1)
gen.add("back_left", double_t, 0, "back left wheel speed multiplier", 1 , -1, 1)
gen.add("back_right", double_t, 0, "back right wheel speed multiplier", 1 , -1, 1)

exit(gen.generate(PACKAGE, "wheel_control", "wheel_control"))
